src/index.html

```diff
@@ -7,6 +7,44 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar"
+      start-day="sat"
+      month-length="29"
+    >
```

Comment:
Per task:

> Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values
> Use [@each](https://sass-lang.com/documentation/at-rules/control/each) to create all the modifiers
> The month should start at the correct column (Monday is the 1st, Friday is the 5th)
> You can just add correct margin-left for the first day
> Add a modifier month-length for the calendar with values 28, 29, 30 and 31 (use @for)
> It sets the last day to show (use [nth-child](https://css-tricks.com/how-nth-child-works/))

That means that you are expected to have something like:
```
<div
      class="calendar calendar--start-days--sat calendar--monght-length--29"
 >
```

While what you are using are HTML attributes

Make sure to repeat corresponding topics if you need to.